v20.2/liquibase.md

```diff
@@ -100,7 +98,7 @@ To install the driver for Liquibase:
 
     {% include copy-clipboard.html %}
     ~~~ shell
-    $ cp ~/Downloads/postgresql-42.2.6.jar liquibase-3.10.2-bin/lib/
+    $ cp ~/Downloads/postgresql-42.2.18.jar liquibase-4.2.0-bin/lib/
```

Comment:
Just checked our CI nightlies, which runs against `42.2.9`. Can we be consistent and updated the pgjdbc recommended version to be in sync?